extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -347,7 +348,8 @@ class AmpLightbox extends AMP.BaseElement {
    * @private
    */
   scrollHandler_() {
-    const currentScrollTop = this.element./*OK*/scrollTop;
+    // If scroll top is 0, it's set to 1 to avoid scroll-freeze issue.
+    const currentScrollTop = this.element./*OK*/scrollTop || 1;
```

Comment:
Inside `scrollHandler`. In your code, `this.pos_` is already set to `1`, I'm just saying that you need to also set `this.element./*OK*/scrollTop` to **actually** change the `scrollTop`